pkg/controllers/disruption/drift.go

```diff
@@ -113,8 +112,8 @@ func (d *Drift) ComputeCommand(ctx context.Context, disruptionBudgetMapping map[
 	return Command{}, scheduling.Results{}, nil
 }
 
-func (d *Drift) Type() string {
-	return metrics.DriftReason
+func (d *Drift) Reason() string {
```

Comment:
```suggestion
func (d *Drift) Reason() v1.DisruptionReason {
```

Rather than returning a string here, can we just return the actual typed value and convert it as-needed